themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -508,9 +513,419 @@ expect(await page.title()).toBe("Pulumi Challenge");
 await browser.close();
 ```
 
+{{% /choosable %}}
+
+{{% choosable cloud gcp %}}
+
+<div class="w-full md:w-1/2">
+  <h3>Prerequisites</h3>
+  <p>In order to complete this challenge, you'll need a couple of things set up in advance.</p>
+  <ul>
+    <li>
+      A <a href="https://app.pulumi.com/signup" target="_blank" rel="noopener noreferrer">Pulumi account</a>
+    </li>
+    <li>
+      The <a href="{{< relref "/docs/get-started/install/" >}}" target="_blank" rel="noopener noreferrer">Pulumi CLI</a>
+    </li>
+    <li>
+      GCP <a href="https://cloud.google.com/gcp" target="_blank" rel="noopener noreferrer">Account</a> & <a href="https://cloud.google.com/resource-manager/docs/creating-managing-projects/" target="_blank" rel="noopener noreferrer">Project</a>
+    </li>
+    <li>
+      <a href="https://www.checklyhq.com/" target="_blank" rel="noopener noreferrer">Checkly</a> account
+    </li>
+  </ul>
+
+  <p><b>Note:</b> New to Google Cloud? You can get $300 in free credits to run, test, and deploy workloads. <a href="https://cloud.google.com/gcp" target="_blank" rel="noopener noreferrer">Open a free GCP account</a> and use it for this Pulumi Challenge</p>
+</div>
+
+### Challenge
+
+Follow along with the steps outlined on this page to complete the challenge.
+
+#### Step 1. Your First Pulumi Program
+
+You will learn how to create a new Pulumi program using our Pulumi templates, specifically for GCP with TypeScript. Create a new directory called `pulumi-challenge` and run the following inside of it, following the prompts if requested:
+
+```shell
+  # set up environment
+  pulumi new gcp-typescript
+
+  # Set up your project id and possibly the region
+  pulumi config set gcp:project <GCP_PROJECT_ID>
+
+  # or whatever your porjectid is
+  pulumi config set gcp:region europe-west1
+```
+
+#### Step 2. Creating Your First Resource
+
+Now that we have a base GCP project configured, we need to create our first resource. In this instance, we’ll create a new [GCS bucket](https://cloud.google.com/storage/docs/buckets) which will allow us to store our static website. The command ```pulumi new gcp-typescript``` produced a a file ```pulumi-challenge/index.ts```. Clear the contents of the ```index.ts``` file and add the following script to create your first resources.
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as gcp from "@pulumi/gcp";
```

Comment:
just to be consistent with the AWS one, you can remove this because this should be included if you ran pulumi new gcp-typescript

Reply:
good - point. will adopt for consistency